_config.yml

```diff
@@ -0,0 +1,8 @@
+theme: jekyll-theme-cayman
+title: Newdle Project
+github_username: indico
+description: > # this means to ignore newlines until "baseurl:"
+  Newdle is an Open Source collaborative tool developed by the Indico Team @ CERN to help you find the perfect date and time for your next meeting in a simple and privacy-preserving way. Newdle is extensible and can be easily integrated in your enterprise ecosystem.
```

Comment:
Check the comment in the first line ;) You do not need to put everything in one super long line. Instead, wrap it after a reasonable length (80 chars maybe?) and remove the comment ;)